_data/trans/dk.json

```diff
@@ -0,0 +1,47 @@
+{
+    "about": "Om",
+    "contribute": "Bidrag på GitHub",
+    "defaultnote_easy": "Besøg den linkede adresse og slet din konto.",
+    "defaultnote_email": "Send en email til den angivne adresse og anmod om at få slettet din konto.",
```

Comment:
Does "send an email" reads the same in Danish?

Reply:
Almost, "send an email" is "send en email" in Danish.